spiffworkflow-backend/src/spiffworkflow_backend/models/task.py

```diff
@@ -56,6 +56,8 @@ class TaskModel(SpiffworkflowBaseDBModel):
     human_tasks = relationship("HumanTaskModel", back_populates="task_model", cascade="delete")
     process_instance_id: int = db.Column(ForeignKey("process_instance.id"), nullable=False, index=True)
 
+    allow_guest: bool = db.Column(db.Boolean())
```

Comment:
could consider killing this cache column if the query filter isn't truly necessary.

Reply:
This has been removed.